cfg/chconf.h

```diff
@@ -130,7 +130,7 @@
  *          infinite loop.
  */
 #if !defined(CH_CFG_NO_IDLE_THREAD)
-#define CH_CFG_NO_IDLE_THREAD               FALSE
+#define CH_CFG_NO_IDLE_THREAD               TRUE
```

Comment:
This creates an extra idle thread. We don't need it because main thread becomes the idle thread.

Reply:
OK